apps/src/p5lab/AnimationPicker/AnimationPreview.jsx

```diff
@@ -134,16 +134,19 @@ export default class AnimationPreview extends React.Component {
     };
 
     const imageStyle = {
-      maxWidth: '100%',
-      maxHeight: '100%',
+      // The maxWidth and maxHeight params need to be 'none' for the animation previews to animate
```

Comment:
huh, this is sort of surprising! how did you find this, and how is it used such that it breaks multi-frame animations? Can you add a little more detail to this comment?

Reply:
Added more details! I decided to fix the animation picker first since I was more familiar with it and got lucky when the AnimationPreview change fixed both the list and the picker 😅

Reply:
thanks thats much more clear and will prevent us from making the same breaking change in the future!